File: profile_service/src/main/java/ru/bmstu/cp/rsoi/profile/model/OperationOut.java
package ru.bmstu.cp.rsoi.profile.model;

import lombok.Data;

@Data
public class OperationOut {
    private String entityName = "user";
    private String parentEntityName = null;
    private String entityId;
    private String parentEntityId = null;
    private String operationName;
    private Long date = System.currentTimeMillis();

    public OperationOut(String entityId, String operationName) {
        this.entityId = entityId;
        this.operationName = operationName;
    }
}

File: session_service/src/main/kotlin/ru/bmstu/cp/rsoi/session/model/OperationOut.kt
package ru.bmstu.cp.rsoi.session.model

data class OperationOut(val entityId: String, val operationName: String) {
    val entityName: String = "user"
    val parentEntityName: String? = null
    val parentEntityId: String? = null
    val date = System.currentTimeMillis()
}